src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainView.scss

```diff
@@ -13,7 +13,8 @@
 
 .statsWrapper {
   padding: 5px 20px 40px 120px;
-  max-width: 1360px;
+  max-width: 1381px;
+  font-size: 13px;
```

Comment:
This is the default font size (see _settings.scss). I tried to disable this rule in the deployment, and didn't see any change. It's probably redundant.

Reply:
Removed it